README.rst

```diff
@@ -33,6 +33,8 @@ Configuration
 - ``NOTES_HOST`` (default: ``"notes.{{ LMS_HOST }}"``)
 - ``NOTES_MYSQL_DATABASE`` (default: ``"notes"``)
 - ``NOTES_MYSQL_USERNAME`` (default: ``"notes"``)
+- ``NOTES_CODE_REPO`` (default: ``"https://github.com/edx/edx-notes-api"``)
```

Comment:
To be consistent with Tutor core, these new settings should be named `NOTES_REPOSITORY` and `NOTES_VERSION`.

Reply:
@regisb using `NOTES_VERSION` will cause a clash with the already existing [version](https://github.com/overhangio/tutor-notes/blob/master/tutornotes/plugin.py#L18) variable which refers to the Tutor plugin version. Should I use something like `NOTES_COMMON_VERSION`?

Reply:
Good point. Can we use "NOTES_REPOSITORY_VERSION" instead please?

Reply:
@regisb addressed. I think this is ready to go